components/esp_lvgl_port/src/lvgl9/simd/lv_color_blend_to_rgb565_esp32s3.S

```diff
@@ -63,6 +64,265 @@ lv_color_blend_to_rgb565_esp:
+            _aligning_mod_8_check_4byte:
+
+            // Check modulo 4 of the unalignment, if - then set 4 bytes
+            bbci a15, 2, _aligning_mod_4_check_4byte     // branch if 2-nd bit unalignment a15 is clear
+                s32i.n      a10,  a3,  0                // save 32 bits from a10 to dest_buff a3, offset 0 bytes
+                addi.n      a3,   a3,  4                // increment dest_buff pointer by 4 bytes
+            _aligning_mod_4_check_4byte:
+
+        _dest_buff_aligned_by_4byte:
+        // Calculate main loop_len
+        srli    a9,    a12,   4                         // a9 - loop_len = local_dest_w_bytes / 16
+
+        // Main loop
+        loopnez  a9, ._main_loop_unaligned_by_4byte     // 16 bytes (8 rgb565) in one loop
+            ee.vst.128.ip q0, a3, 16                    // store 16 bytes from q0 to dest_buff a3
+        ._main_loop_unaligned_by_4byte:
+
+        // Check modulo 8 of the dest_w, if - then set 8 bytes
+        bbci a12, 3, _aligned_mod_8_check_4byte         // branch if 3-rd bit of local_dest_w_bytes a12 is clear
+            ee.vst.l.64.ip    q0,  a3,  8               // save lower 64 bits from q0 to dest_buff a3, increase dest_buff pointer by 8 bytes
+        _aligned_mod_8_check_4byte:
+
+        // Check modulo 4 of the dest_w, if - then set 4 bytes
+        bbci a12, 2, _aligned_mod_4_check_4byte         // branch if 2-nd bit of local_dest_w_bytes a12 is clear
+            s32i.n      a10,  a3,  0                    // save 32 bits from a10 to dest_buff a3, offset 0 bytes
+            addi.n      a3,   a3,  4                    // increment dest_buff pointer by 4 bytes
+        _aligned_mod_4_check_4byte:
+
+        // Check modulo 2 of the dest_w, if - then set 2 bytes
+        bbci a12, 1, _aligned_mod_2_check_4byte         // branch if 1-st bit of local_dest_w_bytes a12 is clear
+            s16i        a10,  a3,  0                    // save 16 bits from a10 to dest_buff a3, offset 0 bytes
+            addi.n      a3,   a3,  2                    // increment dest_buff pointer by 2 bytes
+        _aligned_mod_2_check_4byte:
+
+
+        add     a3,  a3,  a6                            // dest_buff + dest_stride
+        addi.n  a5,  a5,  -1                            // decrease the outer loop
+    bnez a5, .outer_loop_aligned_by_4byte
+
+    movi.n   a2, 1                                      // return LV_RESULT_OK = 1
+    retw.n                                              // return
+
+    _unaligned_by_1byte:
+
+//**********************************************************************************************************************
+
+    // either dest_buff or dest_stride is not 4-byte aligned
+    // dest_w is always 4-byte multiple
+
+    // dest_buff   (a3) - 4-byte, or 1-byte aligned
+    // dest_stride (a6) - 4-byte, or 1-byte multiple
+    // dest_w      (a4) - 4-byte multiple
+
+
+    mov   a13, a3
```

Comment:
Do you need this line? It's a copy-paste form `_argb8888_esp32s3.S` and it looks like I am not using it, just an dead line. You can remove it from both files, if you don't need it either.